### ISSUE TITLE
Train EA-LSTM model using MNTOHA data

### ISSUE THREAD

**Issue by AndyMcAliley** [USGS-R/lake-temperature-lstm-static#20]
Train an EA-LSTM using MNTOHA data. The code for the training loop of the model should be fairly short and straightforward. We'll track loss over epochs. After training, we'll save the parameters of the trained model and the metrics tracked during training.
